publish.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+# Correct dimensions for banner
+CORRECT_B_W=800
+CORRECT_B_H=450
+
+# Correct dimensions for icon
+CORRECT_I_W=128
+CORRECT_I_H=128
```

Comment:
Open to deliberation about this. Could do a multiple, or just make sure that it's square.

Reply:
I think just ensuring that it is square is quite good enough. If you want to change it to that... or i could change it to that... hmmmm